runtime/moonbase/src/precompiles.rs

```diff
@@ -232,6 +233,11 @@ type MoonbasePrecompilesAt<R> = (
 		(CallableByContract, CallableByPrecompile),
 	>,
 	PrecompileAt<AddressU64<2070>, GmpPrecompile<R>, SubcallWithMaxNesting<0>>,
+	PrecompileAt<
+		AddressU64<2071>,
```

Comment:
This address is already reserved for the xcm transactor precompile v3, please increment it